accounting/bank/feeds/ponto.rst

```diff
@@ -31,86 +27,98 @@ Link your bank accounts with Ponto
-platforms.
-
-.. image:: media/ponto_link_odoo.png
-   :align: center
-   :alt: Logo of the Ponto brand
-
-#. To generate these IDs, go to :menuselection:`Ponto Dashboard --> Integration --> Live`, click on
-   *Add Integration*, fill out the form, and select which accounts you want to synchronize.
-#. Copy the Client ID and Secret ID generated and paste them in Odoo.
-#. Configure the synchronization options:
-
-   - **Action**: define if Odoo must create a new *Journal* for this bank account, link to an
-     existing Journal, or create no link.
-   - **Journal**: select the appropriate *Journal*.
-   - **Synchronization Frequency**: define how often Odoo should fetch the bank statements.
+#. Go to :menuselection:`Accounting --> Configuration --> Add a Bank Account`.
+#. Search your institution, make sure to select the right institution. By selecting the
+   institution, you can verify that the third party provider is Ponto.
+#. Click on *Connect* and follow the steps.
+#. At some point, you will have to authorize the accounts you want to access in Odoo.
+   Please select **all the accounts** you want to synchronize. Even the ones coming
+   from other banking institutions.
 
-.. image:: media/ponto_synchronization.png
-   :align: center
-   :alt: Configure the bank synchronization through Ponto on your Odoo database
+   .. image:: media/ponto-select-accounts.png
+      :align: center
+      :alt: Selection of the accounts you wish to synchronize with Odoo.
 
-.. tip::
-   It is good practice to have one Journal per bank account. If you synchronize a single bank
-   account, link it to the existing *Bank* journal. If you have multiple accounts, it is recommended
-   to pick the *Create new journal* option for all additional accounts.
+#. Finish the flow.
 
-.. _ponto-update-credentials:
+.. note::
+   You have to authorize all the accounts you want to access in Odoo but Odoo will filter
+   the accounts based on institution you selected at the second step.
 
 Update your synchronization credentials
 ---------------------------------------
 
 You might have to update your Ponto credentials or modify the synchronization settings.
 
-To do so, activate the :doc:`Developer Mode <../../../general/developer_mode/activate>`, and go to
-:menuselection:`Accounting --> Configuration --> Online Synchronization`.
+To do so, go to :menuselection:`Accounting --> Configuration --> Online Synchronization` and
+select the institution you want to update credentials. Click on *Update Credentials* button
+to start the flow.
+
+During the update, select **all the accounts** you want to synchronize, even the ones coming
+from other banking institutions.
+
+Fetch new accounts
+------------------
+
+You might want to add new online accounts to your connection.
+
+To do so, go to :menuselection:`Accounting --> Configuration --> Online Synchronization` and
+select the institution you want to fetch others accounts. Click on *Fetch Accounts* button
+to start the flow.
```

Comment:
```suggestion
To do so, go to :menuselection:`Accounting --> Configuration --> Online Synchronization` and select 
the institution you want to fetch the other accounts. Click on *Fetch Accounts* button to start the 
flow.
```

"others" c'est "les autres". il faut "the other accounts". C'était déjà dans la review précédente :)

Reply:
désolé :p